fhir-package-source/package-source.rst

```diff
@@ -0,0 +1,83 @@
+.. _packge_source:
+
+FHIR Package Source
+---------------------------
+
+As of SDK version 4.0 you can use a FHIR Package source to resolve FHIR artifacts like ``StructureDefinitions``, ``ValueSets``, and ``CodeSystems`` from one ore multiple FHIR Package.
```

Comment:
"one or more FHIR Package" -> "one or more FHIR NPM  packages"   

I think the term "package" is quite overloaded, so using the qualifier NPM at least once will help to disambiguate.